src/test/lib/ApiTest.cpp

```diff
@@ -4590,6 +4588,7 @@ void QuicTestTlsParam()
                 ), QUIC_STATUS_SUCCESS);
             }
 
+#ifndef _KERNEL_MODE
```

Comment:
What makes this not work in kernel mode, where it did before?

Reply:
Are you sure it worked in kernel before? There's no kernel coverage for this. In a previous iteration, the kernel tests would return `STATUS_NOT_SUPPORTED`.